draft-ietf-tls-tls13.md

```diff
@@ -1483,11 +1486,12 @@ Structure of this message:
            opaque legacy_session_id<0..32>;
            CipherSuite cipher_suites<2..2^16-2>;
            opaque legacy_compression_methods<1..2^8-1>;
-           Extension extensions<0..2^16-1>;
+           [[ Extension extensions<8..2^16-1>; ]]
```

Comment:
Or we could take the 8 (is it only 8?) on the basis that we are describing what a TLS 1.3 client has to send, which always includes key_share or pre_shared_key and signature_algorithms.